app/decorators/resource_decorator.rb

```diff
@@ -77,6 +76,10 @@ def first_creators
 
   private
 
+    def combined_description
+      [try(:description), try(:publisher_statement)].join(' ')
```

Comment:
I wonder if we want to put a couple line breaks before the `publisher_statement`. But that is probably a decision for Seth after he plays around with it.

Reply:
I mentioned that to him, and he seemed ok with the tacking it together. We'll see at retro...